components/TokenSelectionModal.tsx

```diff
@@ -136,7 +137,9 @@ const CustomTokenSelectionItem = ({
     token: tokenAddress,
     chainId: chain?.id,
   });
-  const isBlacklisted = network.tokenBlacklist.get(tokenAddress);
+  const isBlacklisted = blacklistJson.find(
```

Comment:
Can we change the blacklist message from the current 

> This token is blacklisted and cannot be shielded!

to something that explains why, such as:

> This token cannot be shielded because it is a rebase token

or 

> This token is a rebase token and therefore cannot be shielded